Task 2/Task 2.2/Task 2.2/Gameobj.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Task_2._2
+{
+    abstract class Gameobj
+    {
+        private int _x;
+        private int _y;
+        private int _layer;
+        private bool _permeable;
+        private char _skin;
+        private Map _map;
+        public Gameobj(Layers layer, int x, int y, bool permeable, char skin)
+        {
+
+            Skin = skin;
+            this._layer = (int)layer;
+            X = x;
+            Y = y;
+            Permeable = permeable;
+        }
+
+        public int X
+        {
+            get { return _x; }
+            set 
+            {
+                if(value<=0)
+                _x = value; 
+            }
+        }
```

Comment:
So I can set position only if I set it to 0 and _lower_?
And if it's vice versa - it just ignores that?